### PR TITLE
Add class Phalcon\Http\StatusCode

### PR DESCRIPTION
**Pull request by dreamsxin**

### CODE REVIEW

**Comment by sergeyklay** [comment]
:+1:

**Comment by ovr** [comment]
:+1:

**Comment by Green-Cat** [comment]
:+1:

**Comment by KorsaR-ZN** [comment]
Oh, you're faster than me, I was late :)
:+1:

**Comment by SidRoberts** [comment]
Do the constants need to have the "SC_" prefix? And why is the static `_messages` property needed?

**Comment by andresgutierrez** [comment]
We don't add changes directly to master: https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md#pull-request-checklist

**Comment by sergeyklay** [comment]
@dreamsxin Can u please pull to 2.0.0 branch?

File: phalcon/http/statuscode.zep [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	 *
+	 * @param int statusCode
+	 * @return string
+	 */
+	public static function getStatusText(int statusCode)
+	{
+		if typeof self::_messages != "array" {
+			let self::_messages = [
+				// INFORMATIONAL CODES
+				100 : "Continue",
```
Comment (ogarbe) [inline phalcon/http/statuscode.zep]: for better clarity, you should use the const you defined hundred lines upper :)

Comment (dreamsxin) [inline phalcon/http/statuscode.zep]: If really necessary?

**Comment by sergeyklay** [comment]
I agree with @SidRoberts

**Comment by dreamsxin** [comment]
@SidRoberts @sergeyklay Because `CONTINUE` is reserved keyword, so add prefix.

**Comment by KorsaR-ZN** [comment]
I think the prefix must be such:
- 1xx - INFO_
- 2xx - SUCCESS_
- 3xx - REDIRECT_
- 4xx - CLIENT_
- 5xx - ERROR_

**Pull request closed without merging**